plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/active-filters/block.json

```
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"name": "woocommerce/product-filter-active",
 	"version": "1.0.0",
-	"title": "Filter Options",
+	"title": "Active",
```

Should we use the `Active (Experimental)` as before?

```suggestion
	"title": "Active (Experimental)",
```

---

Yeah, let's do it.